go/benchmarks/benchmark_test.go

```diff
@@ -0,0 +1,183 @@
+package benchmarks
```

Comment:
I took the [iavl benchmark test](https://github.com/cosmos/iavl/blob/master/benchmarks/bench_test.go) as base for this one.